### PR TITLE
[1.14] Use containers/conmon

### PR DESCRIPTION
**Pull request by haircommander**
cherry pick of: https://github.com/cri-o/cri-o/pull/2824

to allow easier upgrade of conmon

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -96,6 +96,12 @@ RUN set -x \
        && cp "$GOPATH"/bin/crictl /usr/bin/ \
        && rm -rf "$GOPATH"
 
+# Install conmon
+RUN VERSION=v2.0.0 &&\
```
Comment (saschagrunert) [inline Dockerfile]: Do we want to use a more recent version here?

Comment (haircommander) [reply Dockerfile]: I am thinking a separate PR for that, this is already pretty bloated

Comment (saschagrunert) [reply Dockerfile]: SGTM

**Comment by haircommander** [comment]
the tests are happy enough (I don't think we expect cgroupsv2 to pass here)

PTAL

**Comment by haircommander** [comment]
happy tests, PTAL @umohnani8 @mrunalp @saschagrunert

**Comment by TomSweeneyRedHat** [comment]
LGTM once the tests are happy

**Comment by haircommander** [comment]
the tests are as happy as this PR should get them to be. we don't expect 1.14 to pass cgroupsv2 tests right now

**Pull request merged**